backend/nlp_sent_analysis.py

```diff
@@ -0,0 +1,61 @@
+import string
+
+import matplotlib.pyplot as plt
+from nltk.sentiment.vader import SentimentIntensityAnalyzer
+import matplotlib.pyplot as plt
+
+print('Libraries imported.')
+
+import pandas as pd
+
+
+# Reading the Json file and convert to an list.
+
+
+path = input("Enter the file Path:")
+dataset = pd.read_json(path).values.tolist()
+print('Dataset imported')
+
+
+def clean_dataset(dataset):
+    
+    new_ds = map(lambda x: x[0].lower(),dataset)    
+
+    return list(new_ds)
+
+
+new_ds = clean_dataset(dataset)
+print('Cleaned dataset')
+
+
+
+def sentiment_analyse(sentiment_text):
+#     print('{}: '.format(sentiment_text), end = '')
```

Comment:
I would remove this commented out code

Reply:
I removed it.

Reply:
Looks good, but if you make variable name more clear .For example, dataTest and newDataTest.